### ISSUE TITLE
Form "OggCamp Announcements" mailing list

### ISSUE THREAD

**Issue by JonTheNiceGuy** [oggcamp/OggCamp-Activities#13]
Create on mailman.lug.org.uk & Advertise

First is for notifying people about an upcoming OggCamp event (so we get less of the "I didn't know it was happening") and then to recruit crew. Should have only the following style posts:

* We've announced the date and venue. Get your tickets!
* Do you want to be crew? Get your tickets!
* We've got the scheduled track fixed. Get your tickets!
* Sponsors are on-board! Get your tickets!
* It's next weekend... are you coming? Get your tickets!

Basically, the same stuff with do with the Social Media accounts in the run-up to the event, but not the retweets/reposts and replies.
